lib/src/embedded/compiler/process.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {spawn} from 'child_process';
+import {resolve} from 'path';
+import {Subject} from 'rxjs';
+import {takeUntil} from 'rxjs/operators';
+
+/**
+ * Invokes the Embedded Sass Compiler as a Node child process, exposing its
+ * stdio as Observables.
+ */
+export class EmbeddedProcess {
+  private readonly process = spawn(
+    resolve(__dirname, '../../vendor/dart-sass-embedded'),
+    {
+      windowsHide: true,
+    }
+  );
+
+  private readonly _stdout$ = new Subject<Buffer>();
```

Comment:
I believe using `_` as a prefix violates the Google TS style guide, which we should try to follow here. Maybe `rawStdout$` instead?

Also, it's a good idea to document why this is different than the public property.